base_geoengine/fields.py

```diff
@@ -25,27 +28,140 @@ class GeoField(Field):
 
     geo_type = None
```

Comment:
IMO `geo_type` is no more required. You can directly use _type. Therefore the override of `ir.model.fields` will be no more needed